components/App.js

```diff
@@ -124,12 +124,26 @@ class App extends React.Component {
               alignItems: "center",
               backgroundColor: colors.white,
               width: "100vw",
-              height: this.state.marker ? null : "100vh",
+              
               color: colors.night,
             }}
           >
-            <h1 style={{ paddingTop: "2.7rem" }}>covilla</h1>
-            <div style={{ padding: "0 4rem", textAlign: "center" }}>
+            <h1
+              style={{
+                paddingTop: "2.7rem",
+                fontSize: this.state.marker ? "100%" : "200%",
+                transition: "font-size 0.5s ease-in-out",
+             }}
+            >
+              covilla
+            </h1>
+            <div
+              style={{
+                padding: "0 4rem",
+                textAlign: "center",
+                fontSize: this.state.marker ? "50%" : "100%",
```

Comment:
this makes me think that maybe there should be a single div that contains the h1, Typist div, and MapChart that shrinks, instead of shrinking all three. Then pull `DetailsView` out of `MapChart` and into `main` below it

Reply:
Agreed — I will tackle that next!